app/Console/Commands/ObjectImportCommand.php

```diff
@@ -5,8 +5,8 @@
 use Symfony\Component\Console\Input\InputArgument;
 use Symfony\Component\Console\Input\InputOption;
 
-ini_set('max_execution_time', env('IMPORT_TIME_LIM', 600)); //600 seconds = 10 minutes
-ini_set('memory_limit', env('IMPORT_MEM_LIM', '500M'));
+ini_set('max_execution_time', env('IMPORT_TIME_LIMIT', 600)); //600 seconds = 10 minutes
+ini_set('memory_limit', env('IMPORT_MEMORY_LIMIT', '500M'));
```

Comment:
I'm okay with this change so long as we *never* went to Master with it set as `_LIM` (or `_MEM`) - I don't want a user to do a regular-seeming upgrade and have their stuff break in weird ways because we changed the name of an `.env var.